lms/djangoapps/courseware/views.py

```diff
@@ -1577,12 +1593,14 @@ def financial_assistance_form(request):
     enrolled_courses = [
         {'name': enrollment.course_overview.display_name, 'value': unicode(enrollment.course_id)}
         for enrollment in CourseEnrollment.enrollments_for_user(user).order_by('-created')
-        if CourseMode.objects.filter(
+
+        # Here we are doing a database query as an if condition because if the first condition fails
```

Comment:
Seems like we have a nit conflict :)
https://github.com/edx/edx-platform/pull/11503#discussion_r52821257
@bderusha